packages/tools/viewer/src/viewerElement.ts

```diff
@@ -869,21 +869,43 @@ export abstract class ViewerElement<ViewerClass extends Viewer = Viewer> extends
     /** @internal */
     // eslint-disable-next-line @typescript-eslint/naming-convention
     protected override render() {
+        return html`
+            <div class="full-size">
+                <div id="canvasContainer" class="full-size"></div>
+                ${this.renderOverlay()}
+            </div>
+        `;
+    }
+
+    /** @internal */
+    /**
+     * Renders the progress bar
+     * @returns The template result for the progress bar.
+     */
+    // eslint-disable-next-line @typescript-eslint/naming-convention
+    protected renderProgressBar(): TemplateResult {
         const showProgressBar = this.loadingProgress !== false;
         // If loadingProgress is true, then the progress bar is indeterminate so the value doesn't matter.
         const progressValue = typeof this.loadingProgress === "boolean" ? 0 : this.loadingProgress * 100;
         const isIndeterminate = this.loadingProgress === true;
 
-        const progressBar = html`
+        return html`
             <div part="progress-bar" class="bar loading-progress-outer ${showProgressBar ? "" : "loading-progress-outer-inactive"}" aria-label="Loading Progress">
                 <div
                     class="loading-progress-inner ${isIndeterminate ? "loading-progress-inner-indeterminate" : ""}"
                     style="${isIndeterminate ? "" : `width: ${progressValue}%`}"
                 ></div>
             </div>
         `;
+    }
 
-        // Setup the list of toolbar controls.
+    /** @internal */
+    /**
+     * Renders the toolbar
+     * @returns The template result for the toolbar.
+     */
+    // eslint-disable-next-line @typescript-eslint/naming-convention
+    protected renderToolbar(): TemplateResult {

```

Comment:
I don't think in these cases we actually want to disable the Babylon standard naming conventions. The only reason this is used on some other functions (like `render`) is because those functions are part of the base class from Lit, so we can't change them.
```suggestion
    protected _renderToolbar(): TemplateResult {
```

Reply:
Same for `renderProgressBar` and `renderOverlay`